src/js/dark-theme-toggle.js

```diff
@@ -0,0 +1,55 @@
+/*
+* Dark mode toggle
+*
+* In its current state, it will cause a FOIT.
+* Based on Internet research, the only way to
+* prevent this is to have a server-side language
+* insert the dark-theme class before it begins to
+* render.
+*/
+
+var prefersDarkTheme = localStorage.getItem('darkTheme');
+
+/*
+* localStorage will only store strings
+* This means that if it has the value 'false',
+* It will be truey and incorrectly cause the
+* dark theme to load.
+*/
+if (prefersDarkTheme == 'true') {
+	prefersDarkTheme = true;
+} else {
+	prefersDarkTheme = false;
+}
```

Comment:
```suggestion
/*
 * localStorage will only store strings.
 * This means that if it has the value 'false',
 * it will be truthy and incorrectly cause the
 * dark theme to load.
 */
var prefersDarkTheme = localStorage.getItem('darkTheme') === 'true';
```